tests/integration_tests/openmdao_analysis_base_test.py

```diff
@@ -111,6 +111,8 @@ def test_partials(self):
                 method=self.fd_method,
                 form=self.fd_form,
                 step=self.dh,
+                # TODO: For now we have to skip the solver partials until we implement a proper Jacobian transpose routine
+                excludes=["*.coupling.solver"],
```

Comment:
Does this skip all partial checks for the solver? Are there not some we can still test (e.g residual partials w.r.t DVs and node coodinates)

Reply:
Unfortunately, OpenMDAO doesn't allow for that level of granularity at this time. You can only exclude on a component level

Reply:
Damn, guess we need to get back to #336 at some point soon then